example/geometry/t8_example_geometries.cxx

```diff
@@ -589,6 +607,15 @@ t8_analytic_geom (int level, t8_example_geom_type geom_type)
     t8_cmesh_set_tree_class (cmesh, 0, T8_ECLASS_QUAD);
     snprintf (vtuname, BUFSIZ, "forest_moving_lvl_%i", level);
     break;
+  case T8_GEOM_ANALYTIC_QUAD_TO_SPHERE:
```

Comment:
Please also adjust the position of the new geometry in the switch statement for to keep everything in sync

Reply:
It already is.